awx/plugins/ansible_inventory/backport.py

```diff
@@ -314,12 +315,17 @@ def format_group(group):
         return format_group(top)
 
 
+def which(binary):
```

Comment:
distutils has a utility that does basically this (which we may prefer to use):

from distutils.spawn import find_executable

Reply:
So that means you want to always keep the dep on distutils? Okay.

Reply:
I was going to suggest distutils or whole out copying the python3 shutil implementation (https://github.com/python/cpython/blob/master/Lib/shutil.py#L1096).